README.md

```diff
@@ -899,8 +899,9 @@ End
 
 #### %puts (%-),  %putsn (%=) - put string
 
-`%puts` (put string) and `%putsn` (put string with newline) can be used instead of
-(not portable) echo. Unlike echo, not interpret escape sequences all shells.
+`%puts` (put string) and `%putsn` (put string with newline) can be used instead
+of (not portable) echo. Unlike echo, it does not interpret escape sequences in
+all shells.
```

Comment:
I think I don't understand well the meaning of the last sentence.

Does `%puts` and `%putsn` not interpret escape sequences at all, contrary to the builtin `echo` which interpret them on some shells (but not all) ? (it's the meaning of the current correction)

Or, on the contrary, does those directives interpret escape sequence regardless of the current shell ?

Reply:
Does not interpret escape sequences at all.

----
`echo "\033[1mBOLD\033[m"`

* dash: **BOLD**
* bash: \033[1mBOLD\033[m

`%putsn "\033[31mRED\033[m"`

* all: \033[1mBOLD\033[m
----
`echo -e "\033[1mBOLD\033[m"`

* dash: -e **BOLD**
* bash: **BOLD**

`%putsn "\033[31mRED\033[m"`

* all: -e \033[1mBOLD\033[m
----
`echo -n foo`
* dash: foo (without newline)
* bash: foo (without newline)
* [sh on mac](https://stackoverflow.com/questions/53418600/why-doesnt-echo-n-work-in-shell-on-mac): -n foo (with newline)

`%putsn -n`

* all: -n foo (with newline)